bundle/bundle.go

```diff
@@ -223,3 +223,9 @@ func validateDockerish(s string) error {
 	}
 	return nil
 }
+
+// ActionApplicable is an interface for objects that may apply to a subset of actions
+// e.g., Parameters and Outputs
+type ActionApplicable interface {
```

Comment:
I wonder if it's this library's responsibility to provide the ActionApplicable interface. It's mostly a matter of taste I guess, but I think (with golang at least) it should be the library consumer's responsibility to declare and use this interface, if she really needs it.
One hint is that this interface is not used at all in cnab-go.
WDYT?